docs/strategies/ios.md

```diff
@@ -0,0 +1,12 @@
+# iOS Analysis
+
+The iOS buildtool ecosystem consists of two major toolchains: `Carthage` and `Cocoapods`
+
+| Strategy | Direct Deps | Deep Deps | Edges | Tags |
```

Comment:
I've been removing the `Tags` column from our other docs, as it's ill-defined. Let's remove it here too